uint32 timestamp
uint8 dlc
uint32 arbitration_id
uint8[8] payload
uint8 crc
uint8 counter
bool invalid